stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -21,23 +21,31 @@
 
   private final ResponseBodyFileManager mResponseBodyFileManager;
 
+  private final AsyncPrettyPrinterRegistry mAsyncPrettyPrinterRegistry;
```

Comment:
I need a way to access the registry in responseHeadersReceived method from NetworkEventReporterImpl, and it seems that I could access the registry through peerManager. I am also not so sure where the best place to place the registry. I have asked @jasta to take a look when he is available.